src/test/java/com/camping/admin/steps/BaseSteps.java

```diff
@@ -0,0 +1,20 @@
+package com.camping.admin.steps;
+
+import com.camping.admin.CampingAdminServiceApplication;
+import io.cucumber.spring.CucumberContextConfiguration;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
시나리오마다 동작을 수행해주기 위해서는 cucumber의 `@Before`를 사용하셔야 합니다!

```suggestion
import io.cucumber.java.Before;
```